Alignment/CommonAlignmentProducer/python/TkAlMuonSelectors_cfi.py

```diff
@@ -1,31 +1,37 @@
 import FWCore.ParameterSet.Config as cms
 
-TkAlGoodIdMuonSelector = cms.EDFilter("MuonSelector",
-    src = cms.InputTag('muons'),
-    cut = cms.string('isGlobalMuon &'
-                     'isTrackerMuon &'
-                     'numberOfMatches > 1 &'
-                     'globalTrack.hitPattern.numberOfValidMuonHits > 0 &'
-                     'abs(eta) < 2.5 &'
-                     'globalTrack.normalizedChi2 < 20.'),
-    filter = cms.bool(True)
-)
+from Alignment.CommonAlignmentProducer.alignmentGoodIdMuonSelector_cfi import alignmentGoodIdMuonSelector
+TkAlGoodIdMuonSelector = alignmentGoodIdMuonSelector.clone(src = cms.InputTag('muons'),
+                                                           requireGlobal = cms.bool(True),
+                                                           requireTracker = cms.bool(True),
+                                                           minMatches = cms.int32(1),
+                                                           minMuonHits = cms.int32(0),
+                                                           maxEta = cms.double(2.5),
+                                                           maxChi2 = cms.double(20),
+                                                           filter = cms.bool(True))
```

Comment:
You should be able to make this even simpler by doing
```python
AlGoodIdMuonSelector = alignmentGoodIdMuonSelector.clone(src = 'muons',
                                                           requireGlobal = True,
                                                           requireTracker = True,
                                                           minMatches = 1,
                                                           minMuonHits = 0,
                                                           maxEta = 2.5,
                                                           maxChi2 =20,
                                                           filter = True)
```
Dropping the types should work for all the configurations you have made.

Reply:
Thanks for the suggestion, implemented in the last push.